Standards/scs-XXXX-vN-decision-record-template.md

```diff
@@ -0,0 +1,41 @@
+---
+title: _Descriptive title_
+type: Decision Record
+status: Draft
+track: Global  # | IaaS | Ops | KaaS | IAM
+author: Author <Mail or Github handle>
+date: DD-MM-YYYY
+---
+
+<!---
+This is a template striving to provide a starting point for
+creating a standard or decision record adhering to scs-0001.
+Replace at least all text which is _italic_.
+See https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md
+--->
+
+## Introduction
+
+Introduction
+
+## Terminology
+
+Example (abbr. Ex)
+  This is the description for an example terminology.
+
+## Context
+
+What is the issue that we're seeing that is motivating this decision or change?
```

Comment:
Should this not be called `Motivation` like mentioned [here](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md#sections)

Reply:
Not necessarily, according to @mbuechse we would need to adapt the standard for these changes. This was a step that still needs to be done (thats why the PR is still in Draft)

Reply:
Adapted the standard in a Version 2 in order to reflect the document templates `Standard` and `Decision Record`.